packages/fxa-settings/src/components/BannerNew/index.stories.tsx

```diff
@@ -0,0 +1,178 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Meta } from '@storybook/react';
+import AppLayout from '../AppLayout';
+import { withLocalization } from 'fxa-react/lib/storybooks';
+import BannerNew, { ResendCodeSuccessBanner, ResendLinkSuccessBanner } from '.';
+import { sampleDescription, sampleHeading, sampleCtaText } from './mocks';
+
+// TODO in FXA-10621 - This component is a good candidate for using the controls panel once enabled
```

Comment:
If we could use the Storybook controls panel, we would be able to significantly cut down on the number of stories needed to show all the possible states.

Reply:
Aw thanks ❤️ 

It's not currently working because our storybook package versions are not aligned and there are conflicts that prevent the add-on from working. I spent some time looking at upgrading, but it was a rabbit role. It'll have to be done separately.

Good news is that both CoPilot and ChatGPT seem to do a good job with the tedious task of converting the story format when we get to that point.